### PR TITLE
Added default values to refs

### PR DESCRIPTION
**Pull request by RichardLitt**
License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/refs.go [hunk 1]
```diff
@@ -50,9 +50,9 @@ Note: List all references recursively by using the flag '-r'.
 	},
 	Options: []cmds.Option{
 		cmds.StringOption("format", "Emit edges with given format. Available tokens: <src> <dst> <linkname>."),
-		cmds.BoolOption("edges", "e", "Emit edge format: `<from> -> <to>`."),
-		cmds.BoolOption("unique", "u", "Omit duplicate refs from output."),
-		cmds.BoolOption("recursive", "r", "Recursively list links of child nodes."),
+		cmds.BoolOption("edges", "e", "Emit edge format.").Default("`<from> -> <to>`"),
+		cmds.BoolOption("unique", "u", "Omit duplicate refs from output.").Default(false),
```
Comment (whyrusleeping) [inline core/commands/refs.go]: do we really need to have Default false on every bool? I think thats implied...

File: core/commands/refs.go [hunk 2]
```diff
@@ -50,9 +50,9 @@ Note: List all references recursively by using the flag '-r'.
 	},
 	Options: []cmds.Option{
 		cmds.StringOption("format", "Emit edges with given format. Available tokens: <src> <dst> <linkname>."),
-		cmds.BoolOption("edges", "e", "Emit edge format: `<from> -> <to>`."),
-		cmds.BoolOption("unique", "u", "Omit duplicate refs from output."),
-		cmds.BoolOption("recursive", "r", "Recursively list links of child nodes."),
+		cmds.BoolOption("edges", "e", "Emit edge format.").Default("`<from> -> <to>`"),
```
Comment (whyrusleeping) [inline core/commands/refs.go]: This option is a boolean, you can't use a string for a default value (see test failures)

**Comment by whyrusleeping** [comment]
@RichardLitt ping

**Comment by RichardLitt** [comment]
Thanks! Will get to this; haven't forgotten.

**Comment by RichardLitt** [comment]
Fixed. I think having `Default(false)` is still useful - BoolOptions are not always false, which means they should be specified.

**Comment by whyrusleeping** [comment]
@RichardLitt i think its safe to assume that the default is always false (in go, the default value of any bool is false).

**Comment by whyrusleeping** [comment]
@jbenet thoughts on this?

**Comment by jbenet** [comment]
not sure if they will always be false. makes sense to me but i dont think it's guaranteed. things may change too, so users may opt to specify for sure just in case.

**Comment by whyrusleeping** [comment]
the test failures here appear to be because some of our dockerfile deps no longer work. teamcity is passing because that machine has the early (now broken) layers of the image cached. I'll file an issue to get that fixed but it shouldnt block this PR.

**Pull request merged**